### ISSUE TITLE
Objects with enum types generate invalid Swagger

### ISSUE THREAD

**Issue by MikeRalphson** [kminami/apib2swagger#22]
Given the following API Blueprint excerpt (from http://docs.icons8.apiary.io/api-description-document):

```apib
## TotalV3Item (object)
+ name (string, required) icons style name
+ `api_code` (enum[string], required) - icons style code; in other places it named `plarform` or `platform_api_code`
    + Members
        + `all`
        + `ios7`
        + `win8`
        + `win10`
        + `android`
        + `androidL`
        + `color`
        + `office`
+ total (number, required) the quantity of icons
```

The following is generated

```yaml
  TotalV3Item:
    properties:
      api_code:
        type: enum
      name:
        type: string
      total:
        type: number
    required:
      - name
      - api_code
      - total
    type: object
```

Whereas I would expect:

```yaml
  TotalV3Item:
    properties:
      api_code:
        type: string
        enum:
          - `all`
          - `ios7`
          - `win8`
          - `win10`
          - `android`
          - `androidL`
          - `color`
          - `office
      name:
        type: string
      total:
        type: number
    required:
      - name
      - api_code
      - total
    type: object
```

**Comment by kminami** [comment]
Thank you for reporting!
I fixed this at 0.8.1.

**Comment by MikeRalphson** [comment]
Yes, fix seems to be working, many thanks!
